#! /bin/sh /usr/share/dpatch/dpatch-run
## 008_link-with-g++.dpatch by  <divanov@creditreform.bg>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use g++ to link objects

@DPATCH@

--- firebird-1.5.4.4910/builds/posix/make.rules.orig	2006-11-05 15:37:30.000000000 +0100
+++ firebird-1.5.4.4910/builds/posix/make.rules	2007-02-17 23:05:38.259344867 +0100
@@ -91,17 +91,17 @@
 ifdef UseLibToolForLink
 
 
-    LIB_LINK= libtool $(CC(
-    STATICLIB_LINK = libtool $(CC) -all-static
+    LIB_LINK= libtool $(CXX)
+    STATICLIB_LINK = libtool $(CXX) -all-static
 #    LIB_LINK_OPTIONS = -version-info 0:0:0 -release 1.5.0.0 -rpath /usr/lib
     LIB_LINK_OPTIONS += -version-info 0:0:0 -release 1.5.0.0 $(LIB_LINK_RPATH)$(FirebirdInstallPrefix)/lib
 
-    EXE_LINK = libtool $(CC)
-    STATICEXE_LINK = libtool $(CC) -all-static
+    EXE_LINK = libtool $(CXX)
+    STATICEXE_LINK = libtool $(CXX) -all-static
 else
 
     ifndef LIB_LINK
-	LIB_LINK= $(CC)
+	LIB_LINK= $(CXX)
     endif
     
     STATICLIB_LINK= ar cruvs
